lib/pagy.rb

```diff
@@ -116,6 +118,16 @@ def setup_params_var
     raise VariableError.new(self, :params, 'must be a Hash or a Proc', @params) \
           unless (@params = @vars[:params]).is_a?(Hash) || @params.is_a?(Proc)
   end
+
+  def setup_request_path_var
+    request_path = @vars[:request_path]
+    return if request_path.blank?
+
+    raise VariableError.new(self, :request_path, 'bust be a bare path like "/foo"', request_path) \
+          if !request_path.start_with?('/') || request_path.include?('?')
```

Comment:
I figure it would be pretty easy to use something like `foos_url` instead of `foos_path` when trying to set the request path with Rails URL helpers.  It seems like it would be useful to protect against fully qualified URLs with domain names and URLs or paths with additional params.

Reply:
Small typo ("bust" / "must")

Reply:
`blank?` is rails not ruby!

Reply:
Ah, right!  I saw ActiveSupport in the Gemfile but didn't realize that was only in the test environment.  I'll update this.